modules/custom/d_content/src/Plugin/CKEditorPlugin/CkeditorResponsiveTables.php

```diff
@@ -22,7 +22,7 @@ class CkeditorResponsiveTables extends CKEditorPluginBase implements CKEditorPlu
    * Get path to plugin folder.
    */
   public function getPluginPath() {
-    return drupal_get_path('module', 'd_content') . '/js/plugins/d_responsive_tables';
+    return \Drupal::service('extension.path.resolver')->getPath('module', 'd_content') . '/js/plugins/d_responsive_tables';
```

Comment:
Here we should use DependencyInjection, see https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/dependency-injection-in-plugin-block

In short: implement `ContainerFactoryPluginInterface` and add `public static function create` which will call the constructor and pass `extension.path.resolver` to it.